src/redux/Reducers/login.js

```diff
@@ -5,8 +5,8 @@ let initialState = {
     jwt: "",
   },
   serverAddress: "",
-  isLoggedIn: false,
-  firstLogin: false,
+  isLoggedIn: true,
+  firstLogin: true,
```

Comment:
could you also replace these two with `false`?

Reply:
We do need another solution for local dev. I will forget that again, I can guarantee...

Reply:
Can we use dotenv for that?

Reply:
Yes we can do that. We create two env vars one for logged in and one for first startup. But don't you have the possibility to quickly set up a server? Maybe the next thing I will do is to save the token in the local storage so that you not have to relogin.

Reply:
That would be great! I will try out setting up a local server soon to develop some nice frontend.

Reply:
Great. Do we need the env vars also? If yes feel free to push these changes.